### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by louilinn**
Related issue: https://github.com/CirclesUBI/.github/issues/5

Not sure if something else should be added here?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,57 @@
-# Safe Relay Service
+<div align="center">
+	<img width="80" src="https://raw.githubusercontent.com/CirclesUBI/.github/main/assets/logo.svg" />
+</div>
+
+<h1 align="center">safe-relayer-service</h1>
```
Comment (adzialocha) [inline README.md:5]: `safe-relay-service` :+1: :smile:

File: README.md [hunk 2]
```diff
@@ -1,4 +1,57 @@
...
+
+<div align="center">
+  <!-- Licence -->
+  <a href="https://github.com/CirclesUBI/circles-core/blob/main/LICENSE">
+    <img src="https://img.shields.io/github/license/CirclesUBI/circles-core?style=flat-square&color=%23cc1e66" alt="License" height="18">
+  </a>
+  <!-- CI status -->
+  <a href="https://github.com/CirclesUBI/circles-core/actions/workflows/tests.yml">
+   <img src="https://img.shields.io/github/workflow/status/CirclesUBI/safe-relay-service/Build%20docker%20and%20upload%20to%20registry?color=%2347cccb&label=test&style=flat-square " alt="CI Status" height="18">
+  </a>
```
Comment (adzialocha) [inline README.md:23]: This still addresses the circles-core

File: README.md [hunk 3]
```diff
@@ -1,4 +1,57 @@
...
+    <img src="https://img.shields.io/twitter/follow/circlesubi.svg?label=twitter&style=flat-square&color=%23f14d48" alt="Follow Circles" height="18">
+  </a>
+</div>
+
+<div align="center">
+  <h3>
+    <a href="https://circlesubi.github.io/circles-core/">
+      API Docs
+    </a>
+    <span> | </span>
```
Comment (adzialocha) [inline README.md:39]: This can be removed, for the future we should write some API documentation, but thats another task :smile_cat:

**Review by adzialocha — CHANGES_REQUESTED** [review]
Found some minor issues, otherwise looks good!

File: README.md (hunk 2 above)
Comment (louilinn) [reply README.md]: shouldnt have done this in a hurry :sweat_smile:

**Review by adzialocha — APPROVED** [review]
:sparkles:

**Pull request merged**